verify.js

```diff
@@ -43,22 +43,6 @@ module.exports = function (jwtString, secretOrPublicKey, options, callback) {
     return done(new JsonWebTokenError('jwt must be a string'));
   }
 
-  var parts = jwtString.split('.');
-
-  if (parts.length !== 3){
-    return done(new JsonWebTokenError('jwt malformed'));
-  }
```

Comment:
I think this should be kept here, it refers to the token format, not secret related, there is no need to potentially call the `getSecret` function if the jwt is a malformed